app/models/broadcast_event/pool_released.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# At time of writing, this event type is fired when multiplexed library tubes are passed.
+# BroadcastEvent::LibraryComplete events used to be fired at this time point, however they were
+# redefined to be fired at the 'charge and pass' stage, because that made more sense to the users.
```

Comment:
Same thing, I would only describe the current behaviour in this code, so if there is any change is easier to maintain the coments.